tests/components/axis/test_init.py

```diff
@@ -9,19 +9,6 @@
 from tests.common import MockConfigEntry, mock_coro
 
 
-async def test_setup_device_already_configured(hass):
```

Comment:
YAML config is no longer available for the Axis integration so this test is no longer necessary.